bridge/src/mock.rs

```diff
@@ -325,6 +345,77 @@ impl ConcrateSygmaAsset {
 	}
 }
 
+pub struct SygmaDecimalConverter;
+impl DecimalConverter for SygmaDecimalConverter {
+	fn convert_to(asset: &MultiAsset) -> Option<u128> {
+		match (&asset.fun, &asset.id) {
+			(Fungible(amount), _) => {
+				for (asset_id, decimal) in AssetDecimalPairs::get().iter() {
```

Comment:
Instead of reading it from a global variable, we read it from the input parameter

```suggestion
				for (asset_id, decimal) in DecimalPairs::get().iter() {
```